core/lib/spree/testing_support/factories/factories_linter.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.configure do |config|
+  config.before(:suite) do |example|
```

Comment:
I'd prefer that this was a normal spec. As a `before(:suite)`, this makes it impossible to run a single other spec without running this first. It also makes this not report itself as a passing or failing spec.

Reply:
@jhawthorn Yep, that can be a problem. We could provide a `config.files_to_run_one?` or something along those lines; can't really remember the methods name.